_posts/2021-09-14-geoserver-2-20-rc-released.md

```diff
@@ -0,0 +1,176 @@
+
+* Check system.out.println and printStackTrace statements are not accidentally committed, which can add to logs
+* Cognitive complexity checks, start cleaning up methods that are too complex
+* Use StandardCharsets when possible, rather than String
+* Avoid unnecessary object wrapper creation
+* Use short arrays initializers
+* Work towards consistent style with checks to avoid C style array declarations, add missing @Override annotations, and check java generics are used
+
+This dedication helps provide confidence in the technology we publish.
+
+### WMS
+
+Fixes and improvements:
+
+* [GEOS-4939](https://osgeo-org.atlassian.net/browse/GEOS-4939) Coordinate system ISSUE - S-JTSK Krovak East North (`EPSG: 5514`) - cannot be set up
+* [GEOS-10032](https://osgeo-org.atlassian.net/browse/GEOS-10032) Group Layer in Catalog Mode Hide not in capabilities when unauthenticated
+* [GEOS-10013](https://osgeo-org.atlassian.net/browse/GEOS-10013) Mark invalid error while validating or saving a Style
+* [GEOS-9907](https://osgeo-org.atlassian.net/browse/GEOS-9907) Enable usage of labelPoint function in GetFeatureInfo requests
+* [GEOS-9759](https://osgeo-org.atlassian.net/browse/GEOS-9759) Set Response Cache Headers in LayerGroups
+
+The following functionality has been removed:
+
+* [GEOS-10001](https://osgeo-org.atlassian.net/browse/GEOS-10001) Remove animator and animated GIF support from WMS
+  
+  Use of WPS Animation process is provided as an alternative
+
+## WFS
+
+Fixes and improvements:
+
+* [GEOS-10222](https://osgeo-org.atlassian.net/browse/GEOS-10222) WFS CSV OutputFormat delimiter
+
+## WPS
+
+Fixes and improvements:
+
+* [GEOS-9990](https://osgeo-org.atlassian.net/browse/GEOS-9990) Add GUI and REST API to configure the wps-download module
+* [GEOS-10073](https://osgeo-org.atlassian.net/browse/GEOS-10073) WPS animation download process should report about eventual time mis-matches
+
+## WMTS
+
+Improvements and fixes:
+
+* [GEOS-10008](https://osgeo-org.atlassian.net/browse/GEOS-10008) Have GeoServerTileLayer implementing TileJSONProvider
+* [GEOS-9971](https://osgeo-org.atlassian.net/browse/GEOS-9971) GeoWebCache S3 plugin require AWS creds
+
+
+## INSPIRE Extension
+
+New feature:
+
+* [GEOS-10124](https://osgeo-org.atlassian.net/browse/GEOS-10124) Add Language support to INSPIRE extension
+
+Improvements and fixes:
+
+* [GEOS-10211](https://osgeo-org.atlassian.net/browse/GEOS-10211) Unable to pass INSPIRE validation: Version is mandatory \(WMS\)
+* [GEOS-10192](https://osgeo-org.atlassian.net/browse/GEOS-10192) Inspire extension consistent outputResponse element
+* [GEOS-10141](https://osgeo-org.atlassian.net/browse/GEOS-10141) Inspire extension better error message on language not found
+* [GEOS-10163](https://osgeo-org.atlassian.net/browse/GEOS-10163) Incorrect INSPIRE namespace URI
+
+## OpenSearch EO Extension
```

Comment:
```suggestion
## OpenSearch EO Community Module
```

Reply:
Oh if this is a community module I will just remove the section.

I was careful to filter out community modules, OpenSearch EO must have its own tag